api3-whitepaper.tex

```diff
@@ -430,10 +430,10 @@ \subsection{Airnode features}
 Existing solutions using third-party oracles depend on over-redundancy at the oracle level to cover for this, which results in excessive costs as mentioned in Section~\ref{sec:ineffective-redundancy}.
 API3 envisions each API to only be served by its first-party oracle, which means the redundancy has to be implemented at the level of the individual Airnode.
 The node being fully-serverless enables this to be done easily across different availability zones of a single cloud provider, or even across multiple cloud providers.
-It should also be mentioned that it will be possible to containerize Airnode and operate it on-premises, yet using the serverless version will be recommended for almost all use cases.
+It should also be mentioned that it is possible to operate the containerized version of Airnode on-premises, yet using the serverless version will be recommended for almost all use cases.
 
-Airnode is developed by the founding members of API3 and is now open-sourced\footnote{\url{https://github.com/api3dao/airnode}}.
-The software is feature-complete for the protocol described in Section~\ref{sec:airnode-protocol}, and further development will be funded by API3 in the form of grants.
+Airnode is open-sourced with MIT License and ready to enable first-party oracles\footnote{\url{https://github.com/api3dao/airnode}}.
```

Comment:
```suggestion
Airnode is open-sourced, MIT Licensed, and ready to enable first-party oracles\footnote{\url{https://github.com/api3dao/airnode}}.
```

Reply:
It's the specific license that determines what we mean by open source, so I prefer "open-sourced with *" here

Reply:
Sounds good 👍